types/translation.ts

```diff
@@ -33,6 +33,7 @@ export type DictionaryKey =
   'transactions_list_sent_to' |
   'transactions_list_compost_deposit' |
   'transaction_request_to' |
+  'You_have_prevented_kilos_of_garbage'|

```

Comment:
let's try make a convention for standard naming here where we start with the component name where the text appears and then all lowercase. So this would be: 
```suggestion
'dashboard_you_have_prevented_kilos_of_garbage'  |
```